amethyst_controls/src/bundles.rs

```diff
@@ -76,3 +78,52 @@ where
         Ok(())
     }
 }
+
+/// The bundle that creates a flying movement system.
+/// Note: Will not actually create a moving entity. It will only register the needed resources and systems.
```

Comment:
We should provide a small example of how to make said entity

Reply:
There's the full example I guess, would probably require quite a few lines of code to document it in the docs.